docker/diffusers-doc-builder/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt-get -y update && apt-get install -y bash \
     git-lfs \
```

Comment:
Changes in the Dockerfiles have been built and pushed as Docker containers https://github.com/huggingface/diffusers/actions/runs/18335386344.